Doc/c-api/init.rst

```diff
@@ -7,6 +7,213 @@
 Initialization, Finalization, and Threads
 *****************************************
 
+.. _pre-init-safe:
+
+Before Python Initialization
+============================
+
+In an application embedding  Python, the :c:func:`Py_Initialize` function must
+be called before using any other Python/C API functions; with the exception of
+a few functions and the :ref:`global configuration variables
+<global-conf-vars>`.
+
+The following functions can be safely called before Python is initialized:
+
+* Configuration functions:
+
+  * :c:func:`PyImport_AppendInittab`
+  * :c:func:`PyImport_ExtendInittab`
+  * :c:func:`PyInitFrozenExtensions`
```

Comment:
Why is PyInitFrozenExtensions() in this list?

Reply:
It was a request from @serhiy-storchaka. I don't know the rationale :-)

Reply:
It is called before `Py_Initialize()` in `Python/frozenmain.c` on Windows.